ui/dashboards/src/components/DashboardToolbar/DashboardToolbar.tsx

```diff
@@ -145,17 +146,7 @@ export const DashboardToolbar = (props: DashboardToolbarProps) => {
             <Stack direction="row" spacing={1} marginLeft="auto">
               <TimeRangeControls />
               <DownloadButton />
-              {isBiggerThanSm && (
-                <Button
-                  variant="outlined"
-                  color="secondary"
-                  startIcon={<PencilIcon />}
-                  onClick={onEditButtonClick}
-                  sx={{ marginLeft: 'auto' }}
```

Comment:
I didn't keep this `marginLeft: auto` and as far as I can tell, the toolbar looks the same. Wondering if there was a historical reason it was needed that's no longer relevant.

Reply:
I like it -- the `Stack` above is guaranteeing the spacing, and it's nice to not have automatic margins on components that can be used in multiple places. 

Maybe once upon a time that `Edit` button was by itself in the row and needed it, but I don't know for sure